code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -236,12 +236,9 @@
 
 //Scarves
 
-/obj/item/clothing/accessory/scarf
```

Comment:
I'm not sure if removing this code actually does anything, fix wise. I believe the game will see the other scarves and think "Oh, there's a base type that doesn't change anything" and spawning or selecting /obj/item/clothing/accessory/scarf will spawn a tie.

Reply:
...oooh, riiiight. I'll fix that rn.